client/src/components/ProjectForm.js

```diff
@@ -266,17 +274,39 @@ export default function ProjectForm({
         <Box>
           <Grid container justifyContent="space-evenly" sx={{ my: 3 }}>
             <Grid item xs="auto">
-              <StyledButton
```

Comment:
Hey hey

I think this section based on isLoading looks like a duplication of everything except the child element.

How do you feel about something like this instead?

```js
<StyledButton
                type="submit"
                form="project-form"
                variant={
                  !isEdit ? 'secondary' : !editMode ? 'contained' : 'secondary'
                }
                cursor="pointer"
                disabled={isEdit ? !editMode : false}
              >
                { isLoading ? <CircularProgress /> : Save }
              </StyledButton>